src/applications/vaos/containers/TypeOfAppointmentPage.jsx

```diff
@@ -58,6 +58,7 @@ export class TypeOfAppointmentPage extends React.Component {
   };
 
   goForward = () => {
+    this.props.router.push('/new-appointment/reason-appointment');
     this.props.router.push('/new-appointment/type-of-care');
   };
```

Comment:
`goBack()` and `goForward()` should use the routing helper functions like so:
```
 goBack = () => {
    this.props.routeToPreviousAppointmentPage(this.props.router, pageKey);
  };

  goForward = () => {
    this.props.routeToNextAppointmentPage(this.props.router, pageKey);
  };
```

The routes that each page uses for back and forward are consolidated in `newAppointmentFlow.js`.  Looks like you've already modified that file to account for the flow, so we are good there.  Just need to update the `goBack()` and `goForward()` in this component.

Reply:
Actually, please disregard my comment.  This file shouldn't have been modified at all since it is unrelated to TypeOfReason. If you remove line 61 and re-commit, it should remove this file from the PR.